src/app/hooks/queries/useStacksCollectibles.ts

```diff
@@ -14,28 +14,15 @@ const useStacksCollectibles = () => {
     stxAddress = testAddress;
   }
 
-  function fetchNfts({ pageParam = 0 }): Promise<StacksCollectionList> {
-    if (!stxAddress) {
-      return Promise.reject(new InvalidParamsError('stxAddress is required'));
-    }
-    return getNftCollections(stxAddress, selectedNetwork, pageParam || 0, 150);
-  }
+  const fetchNftCollections = (): Promise<StacksCollectionList> =>
+    getNftCollections(stxAddress, selectedNetwork);
 
-  return useInfiniteQuery(['nft-collection-data', stxAddress], fetchNfts, {
+  return useQuery({
+    enabled: !!stxAddress,
     retry: handleRetries,
-    keepPreviousData: false,
-    getNextPageParam: (lastpage, pages) => {
-      const currentLength = pages
-        .map((page) => page.results)
-        .filter(Boolean)
-        .flat().length;
-      if (currentLength < lastpage.total) {
-        return currentLength;
-      }
-      return false;
-    },
-    refetchOnMount: false,
-    refetchOnWindowFocus: false,
+    queryKey: ['nft-collection-data', stxAddress],
+    queryFn: fetchNftCollections,
+    staleTime: 5 * 60 * 1000, // 5mins
```

Comment:
question: do we want to wait that long here. since the new implementation is light weight than previous one? and probably won't hit the rate limit?

Reply:
Maybe 1min is better?

Reply:
i was thinking about two but since extensions doesn't have a refetch method one may be better, wdyt?